src/mux.js

```diff
@@ -37,12 +37,14 @@ class Mux extends Base {
   /**
    * Mux Constructor
    *
-   * @param {string} accessToken - Mux API Access Token
+   * @param {string|options} accessToken - Mux API Access Token, or a configuration object.
```

Comment:
I'd expect something like `@param {string|object} accessToken`, but I don't know swagger/this stuff as well